sources/models/modeled_types/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "modeled_types"
+version = "0.1.0"
+authors = []
```

Comment:
No authors?

Reply:
I think the models have been contributed by the majority of Bottlerocket developers. I'm not sure I could fairly construct an authors list!